read-fonts/src/tables/variations.rs

```diff
@@ -1371,13 +1375,17 @@ impl VariationRegion<'_> {
     pub fn compute_scalar_f32(&self, coords: &[F2Dot14]) -> f32 {
         let mut scalar = 1.0;
         for (i, axis_coords) in self.region_axes().iter().enumerate() {
-            let coord = coords.get(i).map(|coord| coord.to_f32()).unwrap_or(0.0);
+            let peak = axis_coords.peak_coord.get().to_f32();
+            if peak == 0.0 {
+                continue;
+            }
```

Comment:
Seems like here and above could share a method to iter potentially active regions? - I'm naively assuming that to_fixed isn't typically producing a different zero'ness than to_f32 I suppose

Reply:
If the intent is just filtering out regions where peak is 0, then we can do the check on the 2.14 value so it’s not dependent on type. Will add a helper method.